docs/product-manuals/clients/java-client/job-worker.md

```diff
@@ -0,0 +1,29 @@
+---
+id: job-worker
+title: "Job worker"
+---
+
+## Related Resources
+
+- [Job worker basics](/product-manuals/concepts/job-workers.md)
+
+## The java client's job worker
+
+The java client provides a job worker that takes care of polling for available jobs and allows users to focus on writing code to handle the activated jobs.
+
+Generally, the job worker:
+- polls for jobs before it runs out
```

Comment:
Before it runs out...of jobs? ...on its family after going for a pack of cigarette at the corner store? :thinking: